ci/build_wheel_distributed_ucxx.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_dir="python/distributed-ucxx"
+
+export RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+
+./ci/build_wheel.sh distributed-ucxx ${package_dir}
+
+RAPIDS_PY_WHEEL_NAME="distributed_ucxx_${RAPIDS_PY_CUDA_SUFFIX}" rapids-upload-wheels-to-s3 ${package_dir}/dist
```

Comment:
We can move both line 8 and line 12 into build_wheel.sh (and remove the export), then use `package_name` when setting `RAPIDS_PY_WHEEL_NAME`. The only catch will be replacing the hyphen in the name with an underscore, which we have an example of in line 24 of build_wheel.sh (the line setting `__git_commit__`)

Reply:
Still to-do

Reply:
Grr, I hate that GH has discrepancies on what's shown in the "Files changed" tab (suggested changes) and general comments that do not suggest a change (like this one), I always end up missing stuff.

Now done in https://github.com/rapidsai/ucxx/pull/167/commits/2a8495df31ad1c563a7eba0967f39783006e7cc6